docs/unlockedDID.json

```diff
@@ -1,7 +1,24 @@
 {
   "@context": [
     "https://www.w3.org/ns/did/v1",
-    "https://identity.foundation/EcdsaSecp256k1RecoverySignature2020/lds-ecdsa-secp256k1-recovery2020-0.0.jsonld"
+    "https://identity.foundation/EcdsaSecp256k1RecoverySignature2020/lds-ecdsa-secp256k1-recovery2020-0.0.jsonld",
```

Comment:
seems better to fix these issue in the context, potentially also swiching to a w3id.org redirect

Reply:
#24 should enable removing these inline definitions. (Except for `publicKey` which is legacy and this document should probably be changed to use `verificationMethod` instead.)